cmd/kubectl-testkube/commands/pro/login.go

```diff
@@ -126,7 +127,15 @@ func NewLoginCmd() *cobra.Command {
 
 			common.ProcessMasterFlags(cmd, &opts, &cfg)
 
-			token, refreshToken, err := common.LoginUser(opts.Master.URIs.Auth, opts.Master.CustomAuth, opts.Master.CallbackPort)
+			var token, refreshToken string
+
+			if email != "" {
+				// SSO authentication flow
```

Comment:
I think we can use the SSO flow for all cases

Reply:
The same scenario here, we could but if we want to release it today it doesn't seem feasible..